index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>  
+  <link
+  rel="stylesheet"
+  href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+/>
+  <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+  <main class="home-section">
+    <section class="home">
+      <header>
+        <nav id="bars">
+          <a href="#" class="nav-link" id="menuLink"><i class="fa fa-bars" aria-hidden="true"></i></a>
+        </nav>
+        <div class="header-nav-desktop">
+          <div class="Social-icon">
+            <ul class="Social-icon-list">
+              <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href=""><i class="fab fa-twitter"></i></a></li>
+              <li><a href="">English</a></li>
+              <li><a href="">My Page</a></li>
+              <li><a href="">Logout</a></li>
+            </ul>
+          </div>
+          <div class="nav-item">
+           <a href="index.html"><img src="image/logo.png" alt=""></a> 
```

Comment:
- [ ] Please kindly fix the routes to your different images. The image link generates an error due to the wrong directory name so please change `image` to `Image`.